internal/integrationtest/lib/lib_test.go

```diff
@@ -435,7 +435,7 @@ func TestListWithFqbn(t *testing.T) {
 	require.NoError(t, err)
 	require.Empty(t, stderr)
 	lines := strings.Split(strings.TrimSpace(string(stdout)), "\n")
-	require.Len(t, lines, 2)
+	require.Len(t, lines, 7)
```

Comment:
I would also add a test [like the ones above](https://github.com/arduino/arduino-cli/pull/2000/commits/24b6b7e99f2235a239cc0a915679df43209cf5c4#diff-d32aca018aa49dfc802158743f5c0937da752021b344fdc0236931e334929ae6R428-R431) to test if a lib from the core is currently listed